btrfs/datadog_checks/btrfs/btrfs.py

```diff
@@ -131,10 +132,10 @@ def check(self, instance):
         for device, mountpoint in btrfs_devices.iteritems():
             for flags, total_bytes, used_bytes in self.get_usage(mountpoint):
                 replication_type, usage_type = FLAGS_MAPPER[flags]
-                tags = [
+                tags.extend([
```

Comment:
I think you need to create a new array on each iteration or else you will keep previous iteration values for `usage_type` and `replication_type` in the `tags` array.

Reply:
Good catch! Do:

```
custom_tags = instance.get('tags', [])
...
tags = [
    'usage_type:{0}'.format(usage_type),
    'replication_type:{0}'.format(replication_type),
]
tags.extend(custom_tags)
```

Reply:
Good catch! Do:

```
custom_tags = instance.get('tags', [])
...
tags = [
    'usage_type:{0}'.format(usage_type),
    'replication_type:{0}'.format(replication_type),
]
tags.extend(custom_tags)
```

Reply:
Good catch! Do:

```
custom_tags = instance.get('tags', [])
...
tags = [
    'usage_type:{0}'.format(usage_type),
    'replication_type:{0}'.format(replication_type),
]
tags.extend(custom_tags)
```